### ISSUE TITLE
change web address for application traccar web

### ISSUE THREAD

**Issue by bhrochdi** [tananaev/traccar#1922]
Hello,
I successful to run web application traccar in installed on a dedicated server, assuming www.domaine-name.com, what I want to do is to change the default web address www.domaine-name.com to www.domaine-name.com/app/. I changed the setting in the following way <entry key='web.address'></entry> ** =>** <entry key='web.address'>/app</entry> but it's not working. help me.

**Comment by tananaev** [comment]
Do you already have a website on the domain root? If yes, you can setup proxy to access Traccar via `/app` sub-path.

**Comment by bhrochdi** [comment]
yes I have a website at the domain root. how to setup proxy to access Traccar via sub-path / app ?.

**Comment by tananaev** [comment]
What web server are you using for your main website?

**Comment by bhrached** [comment]
Apache

**Comment by tananaev** [comment]
Then you can use Apache proxy module. Following is the page that is not directly related to your question, but should give you an idea on how to set up a proxy:

https://www.traccar.org/secure-connection/

**Comment by bhrochdi** [comment]
Tank you it works :).
